src/App.tsx

```diff
@@ -20,15 +51,25 @@ export const App: React.FC = () => {
               <TodoFilter />
             </div>
 
+            {isHasError && (
+              <div
+                className="notification is-danger"
+                data-cy="PostsLoadingError"
+              >
+                Something went wrong!
```

Comment:
so you have a real error message in your state, so why wouldn't you show that error?